pkg/io/data.go

```diff
@@ -155,6 +157,31 @@ type jsonCounterEvent struct {
 	Values map[string]float64 `json:"args,omitempty"`
 }
 
+type tempJsonCounterEvent struct {
+	jsonEventCore
+	Values map[string]string `json:"args,omitempty"`
+}
+
+func (ce *jsonCounterEvent) UnmarshalJSON(data []byte) error {
+	t := &tempJsonCounterEvent{}
+	if err := json.Unmarshal(data, t); err != nil {
+		return err
+	}
+
+	ce.jsonEventCore = t.jsonEventCore
+	ce.Values = make(map[string]float64)
+	for k, v := range t.Values {
+		f, err := strconv.ParseFloat(v, 64)
+		if err != nil {
+			return err
+		}
+
+		ce.Values[k] = f
+	}
+
+	return nil
+}
+
```

Comment:
I think this part is from your other pull request, could you remove it from this PR please? :slightly_smiling_face:

Reply:
Sorry about that, I forgot I used master branch for this PR so I accidentally updated it